elasticapm/__init__.py

```diff
@@ -17,6 +17,6 @@
 except Exception as e:
     VERSION = 'unknown'
 
-from elasticapm.base import *
-from elasticapm.conf import *
-from elasticapm.traces import *
+from elasticapm.base import *  # noqa E403
```

Comment:
we import that stuff there to trigger Python to import all the code. Flake8 doesn't like unused imports, so we tell it to please allow it, just this one time